java/src/jmri/jmrit/dispatcher/DispatcherFrame.java

```diff
@@ -1816,7 +1818,8 @@ public AllocatedSection allocateSection(AllocationRequest ar, Section ns) {
                 }
             }
         }
-        //check here to see if block is already assigned to an allocated section;
+        //check here to see if block is already assigned to an allocated section
+        // or crosses and allocated section.
```

Comment:
Should `or crosses and allocated section` be or `crosses an allocated section`?

`an` instead of `and`?